src/main/java/com/conveyal/datatools/manager/controllers/api/UserController.java

```diff
@@ -59,9 +59,13 @@
     private static Logger LOG = LoggerFactory.getLogger(UserController.class);
     private static ObjectMapper mapper = new ObjectMapper();
     private static final String charset = "UTF-8";
-    private static String baseUsersUrl = "https://" + AUTH0_DOMAIN + "/api/v2/users";
-    public static JsonManager<Project> json =
-            new JsonManager<>(Project.class, JsonViews.UserInterface.class);
+    // set the base users URL based on whether the domain is the test environment
+    private static String baseUsersUrl = (
+        AUTH0_DOMAIN.equals("localhost:8089")
+            ? "http://localhost:8089"
+            : "https://" + AUTH0_DOMAIN
+    ) + "/api/v2/users";
```

Comment:
I feel like the test code is leaking into the main code a little bit here. Is there a better way to isolate the test URL or override the default value from the test code?

Reply:
Just pushed some changes that hopefully address this comment.